README.md

```diff
@@ -135,6 +135,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [BI4ALL](https://www.bi4all.pt/en/) [:rocket:](https://www.linkedin.com/company/bi4all/jobs/) | Consulting services with several analytics solutions. | `Lisboa` `Porto` |
 | [Critical Software](https://www.criticalsoftware.com) [:rocket:](https://www.criticalsoftware.com/careers) | Mission and business-critical applications. | `Coimbra` `Lisboa` <br> `Porto` `Tomar` <br> `Vila Real` `Viseu` |
 | [Eyenov](https://eyenov.com) | Computer Vision. Machine Learning. Mobile, desktop and embedded. | `Lisboa` `Remote` |
+| [DareData Engineering](https://daredata.engineering/)| Solving Data Science and Data Engineering problems for startups and enterprises. |`Remote`|
```

Comment:
Does it make sense to link https://www.daredata.engineering/join-the-team as a :rocket: career's page?
Also, the description is a bit long, can you please try to shorten it?